.github/workflows/release-plugins.yaml

```diff
@@ -34,27 +31,41 @@ jobs:
         run: |
           BRANCH=$(git rev-parse --abbrev-ref HEAD) #e.g. release-v1.33
           echo "Release branch: ${BRANCH}"
-          GO111MODULE=off go get -u github.com/openshift-knative/hack/cmd/sobranch
+          
+          cd /tmp
+          git clone https://github.com/openshift-knative/hack.git
+          cd hack
+          go install ./cmd/sobranch
+          cd -
```

Comment:
This might be more idiomatic for actions?

```

      - name: Checkout openshift-knative/hack
        uses: actions/checkout@v4
        with:
          repository: 'openshift-knative/hack'
          ref: 'main'
          path: ./src/github.com/openshift-knative/hack
```

https://github.com/openshift-knative/serverless-operator/blob/3e6b2db518166f5ff76eebd89a2b7c6425b92d3a/.github/workflows/release-generate-ci.yaml#L38-L43

Reply:
I will give it a try

Reply:
Worked in my fork: https://github.com/aliok/backstage-plugins-no-fork-test/actions/runs/9058939716/job/24885515156#step:5:4